AEPAnalytics/Tests/TestHelper/TestUtils.swift

```diff
@@ -0,0 +1,78 @@
+//
+// Copyright 2020 Adobe. All rights reserved.
```

Comment:
change to 2021

Reply:
I am reusing this from the edge extension.